doc/nrf/protocols/wifi/regulatory_certification/wifi_radio_test_sample/wifi_build_instructions.rst

```diff
@@ -18,12 +18,16 @@ Combined build for Radio test and Wi-Fi Radio test
 **************************************************
 
 The combined build configuration builds the binary files for the Wi-Fi Radio test that resides in the application core and the Radio test (short-range) that resides in the network core.
-In the :file:`<ncs_repo>/nrf/samples/wifi/radio_test/multi_domain/prj.conf` file, set :kconfig:option:`CONFIG_SUPPORT_NETCORE_PERIPHERAL_RADIO_TEST` = ``y``.
+In the :file:`<ncs_repo>/nrf/samples/wifi/radio_test/multi_domain/prj.conf` file, set :kconfig:option:`CONFIG_NRF70_SR_COEX` = ``y``,:kconfig:option:`CONFIG_NRF70_SR_COEX_RF_SWITCH=` = ``y``
+and :kconfig:option:`CONFIG_SUPPORT_NETCORE_PERIPHERAL_RADIO_TEST` = ``y``.
 
 .. code-block:: shell
 
    $ west build -p -b nrf7002dk/nrf5340/cpuapp (DK build)
 
+.. note::Add commentMore actions
+   Set ``sr_ant_switch_ctrl 1`` in Wi-Fi Radio test UART port to switch BLE RF signal towards the BLE U.FL socket.
```

Comment:
Ideally this command should be part of the SR/Peripheral radio test (applicable only to nRF7002DK) and not Wi-Fi radio test. We'll deprecate from the Wi-Fi section once this is implemented in the SR radio test.